posthog/test/test_filter_model.py

```diff
@@ -137,6 +137,21 @@ def test_user_properties(self):
         self.assertEqual(events[0], event2)
         self.assertEqual(len(events), 1)
 
+    def test_cohort_properties(self):
+        person1_distinct_id = "person1"
+        person1 = Person.objects.create(team=self.team, distinct_ids=[person1_distinct_id], properties={"group": 1})
+        cohort1 = Cohort.objects.create(team=self.team, groups={}, name="cohort1")
+        cohort1.people.add(person1)
```

Comment:
other tests use `cohort.calculate_people()` so that might be worth changing for consistency

Reply:
talked about it in person but I think we should make our tests more unit-y and reduce dependencies on outside logic like this so that we know when things break, what breaks. Meaning that if you break just this functionality just this test fails making it a lot easier to debug.

Reply:
https://github.com/PostHog/posthog/issues/1786